src/client/tablet_client.cc

```diff
@@ -776,12 +776,7 @@ bool TabletClient::Get(uint32_t tid, uint32_t pid, const std::string& pk, uint64
 
 bool TabletClient::Count(uint32_t tid, uint32_t pid, const std::string& pk, const std::string& idx_name,
                          bool filter_expired_data, uint64_t& value, std::string& msg) {
-    return Count(tid, pid, pk, idx_name, "", filter_expired_data, value, msg);
-}
-
-bool TabletClient::Count(uint32_t tid, uint32_t pid, const std::string& pk, const std::string& idx_name,
-                         const std::string& ts_name, bool filter_expired_data, uint64_t& value, std::string& msg) {
-    ::openmldb::api::CountRequest request;
+     ::openmldb::api::CountRequest request;
```

Comment:
intent, an extra space?

Reply:
Not sure how that happened, guessing it was just an issue with VSCODE not liking my paste. I've updated it and pushed a new commit to fix.